packages/adena-extension/src/repositories/common/chain.ts

```diff
@@ -5,6 +5,66 @@ import { NetworkMetainfo } from '@states/network';
 
 type LocalValueType = 'NETWORKS' | 'CURRENT_CHAIN_ID' | 'CURRENT_NETWORK_ID';
 
+const defaultNetworks: NetworkMetainfo[] = [
+  {
+    id: 'test3',
+    main: true,
+    chainId: 'test3',
+    chainName: 'GNO.LAND',
+    networkId: 'test3',
+    networkName: 'Testnet 3',
+    addressPrefix: 'g',
+    rpcUrl: 'https://rpc.test3.gno.land',
+    gnoUrl: 'https://rpc.test3.gno.land',
```

Comment:
shouldn't it be `https://test3.gno.land` ?

Reply:
I'll fix it.